blocks/identity-block/themes/news.json

```diff
@@ -75,5 +78,75 @@
 			},
 			"desktop": {}
 		}
+	},
+	"sign-up": {
+		"styles": {
+			"default": {
+				"inline-size": "calc(100% - var(--global-spacing-6))",
+				"margin": "auto",
+				"font-family": "var(--font-family-primary)",
+				"max-inline-size": "none",
+				"components": {
+					"button": {
+						"font-size": "var(--global-font-size-4)"
+					},
+					"heading": {
+						"border-block-end-color": "var(--border-color)",
+						"border-block-end-style": "var(--global-border-style-1)",
+						"border-block-end-width": "var(--global-border-width-1)",
+						"font-size": "var(--global-font-size-9)",
+						"margin-block-end": "var(--global-spacing-4)",
+						"margin-bottom": "var(--global-spacing-4)",
+						"padding-block-end": "var(--global-spacing-2)",
+						"text-align": "center"
+					},
+					"input": {
+						"margin-block-end": "var(--global-spacing-5)"
+					},
+					"input-error-tip": {
+						"color": "var(--status-color-danger)"
+					},
+					"input-input": {
+						"inline-size": "100%",
+						"width": "100%",
+						"padding-block-end": "var(--global-spacing-2)",
+						"padding-block-start": "var(--global-spacing-2)",
+						"padding-inline-end": "var(--global-spacing-2)",
+						"padding-inline-start": "var(--global-spacing-2)",
+						"padding-left": "var(--global-spacing-2)"
```

Comment:
Same here, temporary to get the styling to be correct. I can remove these if we're okay for the styles to be off for a bit while we convert to using logical properties.